ui/index.js

```diff
@@ -286,6 +286,15 @@ function setupStateHooks(store) {
     };
   }
 
+  /**
+   * Reload the extension.
+   *
+   * This is used for the `first-install` E2E test, which uses a production-like build. This
+   * function must be present even if `process.env.IN_TEST` is false.
+   */
+  window.stateHooks.reloadExtension = () => {
+    browser.runtime.reload();
+  };
```

Comment:
Just a quite drive by comment: Since we use `browser.runtime.reload()` in a few places in our code, it might make sense to update those to use this new `stateHooks.reloadExtension` line... that way we can say we aren't shipping test functions in production 😆

Reply:
Hmm, interesting idea, but it would probably be better to minimize the reliance on functions we make accessible outside the root compartment. Just to minimize the complexity of any audits into what could go wrong if they're tampered with somehow.